src/Umbraco.Web.UI/umbraco/config/lang/zh.xml

```diff
@@ -1606,6 +1606,7 @@
     <key alias="enableUrlTracker">启用 url 跟踪程序</key>
     <key alias="originalUrl">原始网址</key>
     <key alias="redirectedTo">已重定向至</key>
+    <key alias="redirectUrlManagement">Redirect Url Management</key>

```

Comment:
Can you revert the changes to all the language files please? New keys only need to be in `en.xml` and `en_us.xml`, the latter is used for fallbacks if a key is not available in the user's language.

Reply:
done, I always thought was being helpful adding the extra keys in!!